Dockerfile

```diff
@@ -1,18 +1,24 @@
-FROM alpine:edge
+FROM alpine:latest
```

Comment:
Note that this change will switch from `edge` to the latest supported Alpine release (currently 3.12). It would be wiser to be more explicit and just state `3.12` here and update accordingly in the future when newer releases of Alpine or dnsmasq are made available.